packages/fxa-payments-server/src/components/Survey/index.scss

```diff
@@ -0,0 +1,125 @@
+@import '../../../../fxa-content-server/app/styles/variables';
+
+$shadow: 0 12px 18px 2px rgba(34, 0, 51, 0.04),
+  0 6px 22px 4px rgba(7, 48, 114, 0.12), 0 6px 10px -4px rgba(14, 13, 26, 0.12);
+$survey-height: 360px;
+
+.survey-component {
+  align-items: flex-end;
+  background: $grey-10;
+  border-radius: 4px 4px 0 0;
+  bottom: -360px;
+  box-shadow: $shadow;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  overflow: hidden;
+  position: fixed;
+  right: 10%;
+  transition: bottom 250ms ease-out;
+  width: 320px;
+}
+
+.survey-control {
+  align-items: center;
+  background: 50% no-repeat url(./images/chevron.svg);
+  background-color: transparent;
+  border: 0;
+  display: flex;
+  height: 40px;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+  width: 40px;
+}
+
+.survey-control:focus {
+  box-shadow: none;
```

Comment:
This isn't a must, but you could move some of these pseudo selectors and styles under `.survey-control`.